src/content/blog/2024/05/20/react-conf-2024-recap.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "React Conf 2024 Recap"
+author: Ricky Hanlon
+date: 2024/05/20
+description: Last week we hosted React Conf 2024, a two-day conference in Henderson, Nevada where over 800 attendees gathered to discuss the latest in UI engineering. This was our first in-person conference since 2019, and we were thrilled to be able to bring the community together again.
+---
+
+May 20, 2024 by [Ricky Hanlon](https://twitter.com/rickhanlonii).
+
+---
+
+<Intro>
+
+Last week we hosted React Conf 2024, a two-day conference in Henderson, Nevada where over TODO attendees gathered to discuss the latest in UI engineering. This was our first in-person conference since 2019, and we were thrilled to be able to bring the community together again.
+
+</Intro>
+
+---
+
+At React Conf 2024, we announced the React 19 RC, React Native New Architecture Beta, the experimental release of the React Compiler, and much more. The entire [Day One](https://www.youtube.com/watch?v=T8TZQ6k4SLE) and [Day Two](https://www.youtube.com/watch?v=0ckOUBiuxVY) stream is available online. 
+
+In this post, we'll summarize the talks and announcements from the event.
+
+## Day One {/*day-one*/}
+
+To kick off Day 1, Meta CTO [Andrew "Boz" Bosworth](https://www.threads.net/@boztank) shared a welcome message followed by an introduction by [Seth Webster](https://twitter.com/sethwebster) and our MC [Ashley Narcisse](https://twitter.com/_darkfadr).
+
+In the Day 1 Keynote, [Joe Savona](https://twitter.com/en_JS) shared our goals and vision for React to make it easy for anyone to build great user experiences. [Lauren Tan](https://twitter.com/potetotes) followed with the State of React, which was downloaded over 1 billion times in 2023 and highlighted the work of the React Community to make React, React.
```

Comment:
This reads like the state of react talk was downloaded that many times

Reply:
Fixed